src/index.js

```diff
@@ -2,10 +2,12 @@
 var ga = require('./ga');
 
 
-function analytics(state) {
-  ga('send', 'pageview', {
-    'page': state.path
-  });
+function analytics(state, options) {
+  if (!options) {
```

Comment:
I would change this to: `options = options || {};`

See (http://stackoverflow.com/questions/2851404/what-does-options-options-mean-in-javascript)